scripts/lib/CIME/build.py

```diff
@@ -248,9 +254,8 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
                 if re.search("Current setting for", line):
                     logger.warn(line)
 
-
     # clm not a shared lib for ACME
-    if get_model() != "acme":
+    if get_model() != "acme" and not (buildlist is not None and "lnd" not in buildlist):
```

Comment:
I'm finding the double (triple if you count "None") negatives here hard to parse. I'd find it easier to understand if this was refactored to the following (you should double-check this logic, though):

```python
if get_model() == "acme" or (buildlist is not None and "lnd" not in buildlist):
    pass
else
```